### PR TITLE
Adds Contributor's Guide

### LINKED ISSUE #257: Add Contributors' Guide and CONTRIBUTING.md
(Similar like in GeophysicalFlows.jl)

### PR DESCRIPTION
**Pull request by navidcy**
Closes #257.

### CODE REVIEW

**Comment by navidcy** [comment]
@BrodiePearson this is essentially a copy of the GeophysicalFlows.jl Contributor's Guide that you just reviewed. If you could have a look would be great.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+We follow the [ColPrac guide](https://github.com/SciML/ColPrac) for collaborative
+practices. New contributors should make sure to read that guide.
+
+## What can I do?
+
+* Tackle an existing [issue](https://github.com/FourierFlows/GeophysicalFlows.jl/issues).
+
+* Try to run your favorite GeophysicalFlows.jl module and play around with it to simulate 
+  your particular favorite setup. If you run into any problems or find it difficult
```
Comment (BrodiePearson) [inline CONTRIBUTING.md:19]: your favorite setup.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+* Improve documentation, docstrings, or comments if you found something is hard to use.
+
+* Implement a new feature (e.g., a new diagnostic into a module).
+
+* Implement a new module from scratch to solve your favorite partial differential equation with
+  periodic boundary conditions.
+
+If you're interested in working on something, let us know by commenting on existing issues or 
+by opening a new issue if. This is to make sure no one else is working on the same issue and
```
Comment (BrodiePearson) [inline CONTRIBUTING.md:33]: **by commenting on an existing issue or by opening a new issue.**

**Review by BrodiePearson — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (BrodiePearson) [reply CONTRIBUTING.md:33]: Looks good apart from these two typos!

**Pull request merged**